java/code/src/com/redhat/rhn/frontend/xmlrpc/handler-manifest.xml

```diff
@@ -51,4 +51,5 @@
   <template name="taskomatic.org" classname="com.redhat.rhn.frontend.xmlrpc.taskomatic.TaskomaticOrgHandler" />
   <template name="user" classname="com.redhat.rhn.frontend.xmlrpc.user.UserHandler" />
   <template name="user.external" classname="com.redhat.rhn.frontend.xmlrpc.user.external.UserExternalHandler" />
+  <template name="admin.monitoring" classname="com.redhat.rhn.frontend.xmlrpc.admin.AdminMonitoringHandler"/>
```

Comment:
When this should be not just `admin` i would also name the java package `admin.monitoring`

Reply:
Good point, moved to `admin.monitoring` sub-package.